apiserver/plane/app/views/__init__.py

```
@@ -182,6 +182,10 @@
     IssueCustomPropertyUpdateAPIView,
 )
 
+from .issue_type.issue_type_custom_property import (
+    IssueTypeCustomPropertyAPIEndpoint,
```

Ruff (0.11.9)

`.issue_type.issue_type_custom_property.IssueTypeCustomPropertyAPIEndpoint` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)